static/src/stylesheets/module/facia/_container--video.scss

```diff
@@ -43,6 +43,8 @@ $video-width-desktop: 700px;
         left: 0;
         right: 0;
         bottom: 0;
+        pointer-events: none;
```

Comment:
This is scoped to the `.video-overlay` element, so no need there.

Also selection is essentially disabled on all our items on fronts, due to the `faux-link` overlay.